Common/Vulkan/VulkanContext.cpp

```diff
@@ -1076,7 +1078,11 @@ bool GLSLtoSPV(const VkShaderStageFlagBits shader_type,
 	}
 
 	// Can't fail, parsing worked, "linking" worked.
-	glslang::GlslangToSpv(*program.getIntermediate(stage), spirv);
+	glslang::SpvOptions options;
+	options.disableOptimizer = false;
+	options.optimizeSize = false;
```

Comment:
I wonder if it's ever a good idea to set this to true - also see:
https://github.com/KhronosGroup/glslang/issues/1298

Seems like if we want to optimize, we should use something external (maybe a good idea?) or at least turn on optimizeSize.

-[Unknown]